bigframes/ml/llm.py

```diff
@@ -1009,7 +1013,7 @@ def score(
             "text_generation", "classification", "summarization", "question_answering"
         ] = "text_generation",
     ) -> bpd.DataFrame:
-        """Calculate evaluation metrics of the model. Only "gemini-pro" model is supported for now.
+        """Calculate evaluation metrics of the model. Only "gemini-pro" and "gemini-1.5" models are supported for now.
```

Comment:
same, more explicit to "gemini-1.5-pro-002" and "gemini-1.5-flash-002"

Reply:
done